dataland-inbound-proxy/nginxdev.conf

```diff
@@ -0,0 +1,31 @@
+events {
+
+}
+
+http {
```

Comment:
Wir könnten eigentlich auf das File verzichten:
-> die Dummyserver exposen ihre Ports einfach nach außen
-> die entsprechenden URLs können wir in der application-development.properties im backend hinterlegen
-> den inbound-nginx brauchen wir im docker-compose dann im profil development gar nicht mehr.

Das macht das gesamte Setup nochmal deutlich einfacher.

Reply:
das file wurde wie besprochen entfernt und die profile und ports in der compose file angepasst

Reply:
super - ist jetzt einfacher!